### PR TITLE
you can loosen straps of a sec helmet to show hair

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
title
## Why It's Good For The Game
gives more customization options for security players
## Changelog
:cl: grungussuss
qol: security helmet straps can be loosened to show hair
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -68,6 +68,17 @@
...
 
+/obj/item/clothing/head/helmet/sec/attack_self(mob/user)
+	. = ..()
+	if(.)
+		return
+	balloon_alert(user, "[flags_inv & HIDEHAIR ? "loosening" : "tightening"] straps...")
+	if(!do_after(user, 3 SECONDS, src))
+		return
+	flags_inv ^= HIDEHAIR
+	balloon_alert(user, "[flags_inv & HIDEHAIR ? "tightened" : "loosened "] straps")
```
Comment (Fikou) [inline code/modules/clothing/head/helmet.dm:79]:
```suggestion
	balloon_alert(user, "[flags_inv & HIDEHAIR ? "tightened" : "loosened "]straps")
```

**Comment by optimumtact** [comment]
What does this look like in game? It's important people are able to identify when you are wearing a helmet and we already had a huge argument when we removed the HOS dermal patch for being invisible and settled on armour needing to be visible on sprites.

So if this removes the hat sprite it should not be merged.

**Comment by Sadboysuss** [comment]
> What does this look like in game? It's important people are able to identify when you are wearing a helmet and we already had a huge argument when we removed the HOS dermal patch for being invisible and settled on armour needing to be visible on sprites.
> 
> So if this removes the hat sprite it should not be merged.


https://github.com/user-attachments/assets/36cb776f-087d-4902-9a8f-6782676bdea4

looks a bit silly for some hairs but you can definetly still see the helmet as it layers above the hair

**Comment by Loafin34** [comment]
we have berets for this

**Comment by necromanceranne** [comment]
Might be a good idea for it to still use this to avoid that goofy ass hair sticking out the top https://github.com/tgstation/tgstation/pull/87263

**Comment by carlarctg** [comment]
bad PR, sacrificing armor for drip is an important part of sec balance. removes a lot of the impact of security looking like goons

**Comment by MatrixOne500** [comment]
thank you for making this PR, I felt discouraged to play my female static in sec because the helmets looked ugly and 'baldified' you.

**Comment by SmArtKar** [comment]
I have mixed feelings on this, on one hand its neat and looks cool but on the other hand secoffs no longer look like nameless goons

**Comment by Sadboysuss** [comment]
Uploading 2024-11-04 22-16-35.mp4…

**Comment by Sadboysuss** [comment]
>  look like nameless goons

It's optional and people can use berets and caps already

**Comment by carlarctg** [comment]
> > look like nameless goons
> 
> It's optional and people can use berets and caps already

those have less armor and on purpose

**Comment by Sadboysuss** [comment]
> those have less armor and on purpose

That's what I thought, look again

**Review by SmArtKar — APPROVED** [review]

**Pull request merged**